DungeonApi/DungeonApi/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for DungeonApi project.
+
+Generated by 'django-admin startproject' using Django 3.0.8.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'jo%8!x(b%xmmsv#n)t0=oy($3y#m^4s6(8a4nqc!t1jea&3b1j'
```

Comment:
Would we want to add this into an .env file before we deploy?

Reply:
Thanks for reminding about including the env